src/content/blog/2024/reflections-on-nextflow-mentorship.md

```diff
@@ -0,0 +1,78 @@
+
+# Life after the mentorship
+
+With the skills I acquired during the mentorship as a mentee, I proposed and successfully implemented a custom solution in Nextflow for a precision medicine start-up I worked at the time that could sequentially do several diagnostics and consumer-genetics applications in the cloud, resulting in substantial cost savings and increasing flexibility for the company.
+Beyond my immediate projects, I joined a group actively developing an open-source Nextflow pipeline for genetic imputation. This project allowed me to be in close contact with members of the nf-core community working on similar projects, adding new tools to this pipeline, giving and receiving feedback, and continuing to improve my overall Nextflow skills while also contributing to the broader bioinformatics community. You can learn more about this project with the fantastic talk by Louis Le Nézet at Nextflow Summit 2023 Nextflow Summit 2023 - Louis Le Nézet
+Finally, I was honored to become a Nextflow ambassador. The program’s goal is to extend the awareness of Nextflow around the world while also building a supportive community. In particular, the South American community is underrepresented, so I serve as a point of contact for any institution or newcomer who wants to implement pipelines with Nextflow.
+As part of this program, I was invited to speak at the second Chilean Congress of Bioinformatics, where I gave a talk about how Nextflow and nf-core can support scaling bioinformatics projects in the cloud. It was incredibly rewarding to introduce Nextflow to a community for the first time and witness the genuine enthusiasm it sparks among students and attendees for the potential in their research projects.
+
+<div style="margin-top: 2rem; margin-bottom: 2rem;">
+    <img src="/img/blog-2024-04-10-img1c.png" alt="Second Chilean Congress of Bioinformatics" />
+</div>
+
+# What’s next?
+
+The comprehensive skill set acquired in my journey resonated positively and allowed me to join the ZS Discovery Team. This organization accelerates transformation in research and early development with direct contribution to impactful bioinformatics projects with a globally distributed, multidisciplinary talented team.
+
+In addition, we organized the first Nextflow Hackathon in Argentina, fostering a space to advance our skills in workflow management collectively. It was a pleasure to see how beginners got their first PRs approved and how they interacted with the nf-core community for the first time.
```

Comment:
Was this a local site as part of the nf-core hackathon, or a different thing? Would clarify

Reply:
Yes, it was a site for the nf-core hackathon. I will clarify in the text.